capi-configuration/3_upgrades.md

```diff
@@ -0,0 +1,52 @@
+# Handling workload cluster upgrades
+
+## Problem Statement
+
+We want to ...
+1. Make all changes transparent for the user before application
+2. Continue supporting the current workflow of upgrading a cluster (Through GS staff, customers and automation)
```

Comment:
Can we define a bit more closely what we mean here? Like the one button upgrade, timed upgrade and so on?